### PR TITLE
Next

### PR DESCRIPTION
**Pull request by mhartington**
This PR introduces several new features to angular toolkit. 

1) bumps the minimum version support to Angular v15
2) support for generating standalone pages
3) updates the page template to use the large header/title layout

### CODE REVIEW

File: packages/schematics/page/route-utils/index.ts [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+  const pathToCheck = (options.path || '') + (options.flat ? '' : '/' + strings.dasherize(options.name));
+  let dir: DirEntry | null = host.getDir('/' + pathToCheck);
+  const routesRe = /.routes\.ts/;
+
+  while (dir) {
+    const matches = dir.subfiles.filter((p) => routesRe.test(p));
+    if (matches.length === 1) {
+      return join(dir.path, matches[0]);
+    } else if (matches.length > 1) {
+      throw new Error('whoops');
```
Comment (liamdebeasi) [inline packages/schematics/page/route-utils/index.ts:213]: Can we add a more descriptive error message here?

File: packages/schematics/page/route-utils/index.ts [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    }
+    dir = dir.parent;
+  }
+  throw new Error('Could not find your routes file. Use the skip-import ' + 'option to skip importing.');
```
Comment (liamdebeasi) [inline packages/schematics/page/route-utils/index.ts:217]: Why do we do ` ' + '`?

File: packages/schematics/page/route-utils/index.ts [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+  const routingModuleRe = /-routing\.module\.ts/;
+
+  while (dir) {
+    const matches = dir.subfiles.filter((p) => routingModuleRe.test(p));
+
+    if (matches.length === 1) {
+      return join(dir.path, matches[0]);
+    } else if (matches.length > 1) {
+      throw new Error(
+        'More than one module matches. Use skip-import option to skip importing ' +
```
Comment (liamdebeasi) [inline packages/schematics/page/route-utils/index.ts:54]: Why do we do ` ' + '`?

**Review by liamdebeasi — CHANGES_REQUESTED** [review]

**Comment by mhartington** [comment]
>Why do we do  ' + '?

🤷  Some of this code predates me maintaining angular-toolkit, but I think it's safe to remove.

**Review by liamdebeasi — DISMISSED** [review]
Code looks good, but I added Sean to take a look too.

Only final thing is can we add a write-up for the PR as to what it does/why it's needed?

**Comment by mhartington** [comment]
Rebased and cleaned up the commits so we have can have 3 separate commits that are clear and understood. This should not be squashed, but merged in the traditional way

**Review by liamdebeasi — APPROVED** [review]

**Review by sean-perkins — APPROVED** [review]

**Pull request merged**